src/IO/processFiles.js

```diff
@@ -60,27 +49,29 @@ export const processFiles = async (
     labelImageNames,
     use2D,
   })
-  viewerConfig.config = config
-  viewerConfig.rotate = rotate
-  return createViewer(container, viewerConfig)
+  return createViewer(container, {
+    ...viewerConfig,
+    config,
+    rotate,
+    ...rest,
+  })
 }
 
 export const readFiles = async ({
   files,
   image,
   labelImage,
   labelImageNames,
-  rotate,
   use2D,
 }) => {
   let readDICOMSeries = readImageDICOMFileSeries
-  if (files.length < 2 || !!!image) {
+  if (files.length < 2 || !image) {
     readDICOMSeries = function() {
       return Promise.reject('Skip DICOM series read attempt')
     }
   }
   try {
-    const { image: itkImage, webWorkerPool } = await readDICOMSeries(files)
```

Comment:
I think we need to keep `webWorkerPool` around, also call `webWorkerPool.terminate()`, correct?

Reply:
Ah yes.  Good catch.